### ISSUE TITLE
Remove "Skip to Nav" Link

### ISSUE THREAD

**Issue by aparkening** [alistapart/AListApart#173]
Andrew recommends we remove our "skip to nav" link, as a screen reader can't read it and it isn’t keyboard focusable (uses display:none).

This is a WCAG 2.0 2.4.7 failure (where no focus is visible)

**Comment by murtaugh** [comment]
I wanted to ask @awkawk: Until we move the nav to the top, shouldn't we restyle this instead of removing it? Restyle it to be invisible, but screen-reader accessible, I mean.

**Comment by awkawk** [comment]
Options:
1) Leave it and make it screen reader accessible. This also opens the possibility of measuring whether anyone every clicks on this link, which would be interesting.
2) Leave it and not only make it SR accessible but make it work for keyboard users also (possibly it appears when the previous item is blurred due to tabbing so the focus lands on it only when a keyboard user might need it.  Also allows for measurement.
3) Remove it  -  Screen reader users can access a list of regions (ctrl+insert+r in JAWS, NVDA+F7 in NVDA) so this may not be needed. We haven't had any complaints (have we?) and we're planning to fix the structure so there is an argument to be made that if anyone needs it it is sighted keyboard-only users.

**Comment by murtaugh** [comment]
Ok cool. I’m fine removing it if that’s your preference, just wanted to make sure!

**Comment by murtaugh** [comment]
Ahh, actually, @awkawk, @mikepick just reminded me that the Skip to Nav link actually serves as the hook for our hamburger menu.

So, I think I will go with option #1, and we will visit making it keyboard accessible when we do the same for the nav.

Does that work?

**Comment by awkawk** [comment]
Sure.
